<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="OpenFile">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="PathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>991</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFile">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="SPathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>991</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>1061</width>
      <height>461</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1059</width>
       <height>459</height>
      </rect>
     </property>
     <widget class="QLabel" name="ImgLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1061</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="Equalization">
       <property name="text">
        <string>Equalization</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Histogram">
       <property name="text">
        <string>Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RY">
       <property name="text">
        <string>Reflection Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RX">
       <property name="text">
        <string>Reflection X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Negative">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Negative</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="brightLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>[-128 , 127]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Brightness">
         <property name="text">
          <string>Brightness</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="contrastLabel">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>[0.0 , ...]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Contrast">
         <property name="text">
          <string>Contrast</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="Degree">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>[-360 , 360]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Rotate">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="threshold">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>[0 , 255]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Thresholding">
         <property name="text">
          <string>Thresholding</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="newWidth">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newHeight">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Scale">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="Interpolation">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Interpolation Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="leftMargin">
            <number>40</number>
           </property>
           <item>
            <widget class="QRadioButton" name="Nearest">
             <property name="text">
              <string>Nearest</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Bilinear">
             <property name="text">
              <string>Bilinear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QComboBox" name="Dim">
         <item>
          <property name="text">
           <string>3x3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5x5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7x7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9x9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11x11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13s13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15x15</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="FilterType">
         <item>
          <property name="text">
           <string>Box</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gauss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sharpening</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prewitt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sobel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ApplyFilter">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="Resize">
           <property name="text">
            <string>Auto adjust</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ZoomIn">
           <property name="text">
            <string>Zoom in</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ZoomOut">
           <property name="text">
            <string>Zoom out</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="InfoLabel1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dim:</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeightLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AuxLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WidthLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bits per pixel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="BppLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Size:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>91</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="Undo">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Redo">
       <property name="text">
        <string>Redo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ImgSizeLabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>270</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel5">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bytes</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>PathLabel</zorder>
   <zorder>SaveFile</zorder>
   <zorder>SPathLabel</zorder>
   <zorder>scrollArea</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>OpenFile</zorder>
   <zorder>InfoLabel1</zorder>
   <zorder>NameLabel</zorder>
   <zorder>InfoLabel2</zorder>
   <zorder>HeightLabel</zorder>
   <zorder>AuxLabel</zorder>
   <zorder>WidthLabel</zorder>
   <zorder>InfoLabel3</zorder>
   <zorder>BppLabel</zorder>
   <zorder>InfoLabel4</zorder>
   <zorder>ImgSizeLabel</zorder>
   <zorder>InfoLabel5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
